src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -65,41 +70,94 @@ class Widget_List extends Widget_Abstract {
 	];
 
 	/**
-	 * @todo update in TEC-3612 & TEC-3613
-	 *
 	 * {@inheritDoc}
-	 *
-	 * @var array<string,mixed>
 	 */
-	protected $validate_arguments_map = [
-		'should_manage_url'    => 'tribe_is_truthy',
-		'no_upcoming_events'   => 'tribe_is_truthy',
-		'featured_events_only' => 'tribe_is_truthy',
-		'jsonld_enable'        => 'tribe_is_truthy',
-	];
+	protected function setup_default_arguments() {
+		$default_arguments = parent::setup_default_arguments();
+
+		$default_arguments['description'] = esc_html_x( 'A widget that displays upcoming events.', 'The description of the List Widget.', 'the-events-calendar' );
+		// @todo update name once this widget is ready to replace the existing list widget.
+		$default_arguments['name']                          = esc_html_x( 'Events List V2', 'The name of the widget.', 'the-events-calendar' );
+		$default_arguments['widget_options']['description'] = esc_html_x( 'A widget that displays upcoming events.', 'The description of the List Widget.', 'the-events-calendar' );
+		// Setup default title.
+		$default_arguments['title'] = _x( 'Upcoming Events', 'The default title of the List Widget.', 'the-events-calendar' );
+
+		return $default_arguments;
+	}
 
 	/**
-	 * @todo update in TEC-3612 & TEC-3613
-	 *
 	 * {@inheritDoc}
 	 */
-	public function setup() {
-		$this->setup_view();
+	public function update( $new_instance, $old_instance ) {
+		$instance = $old_instance;
+
+		/* Strip tags (if needed) and update the widget settings. */
+		$instance['title']                = wp_strip_all_tags( $new_instance['title'] );
+		$instance['limit']                = $new_instance['limit'];
+		$instance['no_upcoming_events']   = ! empty( $new_instance['no_upcoming_events'] );
+		$instance['featured_events_only'] = ! empty( $new_instance['featured_events_only'] );
+		$instance['jsonld_enable']        = (int) ( ! empty( $new_instance['jsonld_enable'] ) );
+
+		return $this->filter_updated_instance( $instance );
 	}
 
 	/**
 	 * {@inheritDoc}
 	 */
-	public function get_arguments() {
-		$arguments = $this->arguments;
+	public function get_admin_fields() {
 
-		$arguments['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
-		// @todo update name once this widget is ready to replace the existing list widget.
-		$arguments['name']                          = esc_html__( 'Events List V2', 'the-events-calendar' );
-		$arguments['widget_options']['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
+		return [
+			'title'                => [
+				'label' => _x( 'Title:', 'The label for the field of the title of the List Widget.','the-events-calendar' ),
```

Comment:
:no_entry_sign: No space found after comma in argument list

<sub>hash: 49132dfd92873a9faa3736fe405e6ded</sub>